frontend/app/helpers/spree/products_filters_helper.rb

```diff
@@ -42,8 +45,70 @@ def price_filter_input(name:, value:, placeholder:, **html_options)
       )
     end
 
+    def option_value_filter_link(option_value, permitted_params, opts = {})
```

Comment:
@mad-eel we should also allow here via `opts` to switch CSS classes and other stuff